Source/WebCore/platform/graphics/cocoa/UnrealizedCoreTextFont.h

```diff
@@ -70,16 +71,19 @@ class UnrealizedCoreTextFont {
 private:
     CGFloat getSize() const;
 
-    enum class OpticalSizingType : uint8_t {
-        None,
-        JustVariation,
-        Everything,
+    struct OpticalSizingTypes { // Ideally this would be a namespace, but clang doesn't seem to let you define a namespace inside a class.
+        // When USE(CORE_TEXT_OPTICAL_SIZING_WORKAROUND) is no longer necessary, we can migrate this back to an enum.
+        struct None { };
+        struct JustVariation { };
+        struct Everything { std::optional<float> opticalSizingValue; };
```

Comment:
Could we save some memory by using a `Markable` instead of an `optional` here ? (do we actually need the 0 value of the negative values?)

Reply:
I think we actually do need the full range, because the value that this gets set to is supplied by CSS (parsed via `CSSPropertyParserHelpers::consumeNumberRaw()`)